config/sync/config_pages.type.lockup_settings.yml

```diff
@@ -0,0 +1,56 @@
+uuid: a18599df-51cc-4b9e-96d0-7a127d0247ed
+langcode: en
+status: true
+dependencies:
+  module:
+    - config_pages_overrides
+third_party_settings:
+  config_pages_overrides:
+    cc9d47c1-ac6c-426d-b3d0-b6b58bd31a51:
+      field: su_lockup_options
+      delta: 0
+      column: value
+      config_name: stanford_basic.settings
```

Comment:
Is this currently only supporting stanford_basic?

Reply:
yes

Reply:
After this missed the last release you were asked to include multiple theme support in our sprint planning meeting but I don't see it written down anywhere so I suspect we forgot. I won't block this PR for that requirement but can you add a link to/create the ticket for the multiple theme support to this PR?

Reply:
will do